src/compas/geometry/xforms/quaternion.py

```diff
@@ -0,0 +1,118 @@
+from compas.geometry.transformations import quaternion_multiply
+from compas.geometry.transformations import quaternion_conjugate
+from compas.geometry.transformations import quaternion_unitize
+from compas.geometry.transformations import quaternion_canonic
+from compas.geometry.transformations import quaternion_norm
+from compas.geometry.transformations import quaternion_is_unit
+
+__all__ = ['Quaternion']
+
+class Quaternion(object):
+    def __init__(self, w,x,y,z):
```

Comment:
leave a blank line and add a docstring

Reply:
added. 
Since the `@property`  methods are not in the docs, where will the examples within them appear?